The Flask and FastAPI framework (seminars)/Урок 2. Погружение во Flask/templates/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Форма ввода данных</title>
+</head>
+<body>
+    <h1>Введите имя и электронную почту</h1>
+    <form action="/submit" method="post">
```

Comment:
Тогда можно не указывать атрибут `action`
```suggestion
    <form method="post">
```